iocore/net/YamlSNIConfig.h

```diff
@@ -26,25 +26,32 @@
 
 #include "tsconfig/Errata.h"
 
-constexpr char TS_fqdn[]                 = "fqdn";
-constexpr char TS_disable_H2[]           = "disable_h2";
-constexpr char TS_verify_client[]        = "verify_client";
-constexpr char TS_tunnel_route[]         = "tunnel_route";
-constexpr char TS_verify_origin_server[] = "verify_origin_server";
-constexpr char TS_client_cert[]          = "client_cert";
-constexpr char TS_ip_allow[]             = "ip_allow";
+#define TSDECL(id) constexpr char TS_##id[] = #id
```

Comment:
How could you do this to me?

Reply:
This was @ywkaras's suggestion. Eliminates errors in double typing